### PR TITLE
New site: WALTON-CLOUD

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Brings a new site to the cloud-info-provider: [WALTON-CLOUD](https://goc.egi.eu/portal/index.php?Page_Type=Site&id=3392)

---

**Related issue :**

### CODE REVIEW

File: sites/WALTON-CLOUD.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+gocdb: WALTON-CLOUD 
+endpoint: https://waltoncloud.eu:5000/v3
```
Comment (gwarf) [inline sites/WALTON-CLOUD.yaml:3]: https://horizon.waltoncloud.eu:5000/v3 ?

Comment (enolfc) [reply sites/WALTON-CLOUD.yaml]: fixed!

**Comment by enolfc** [comment]
Missing an ops project that we can reference here

**Comment by gwarf** [comment]
> Missing an ops project that we can reference here

For which purpose exactly?

**Comment by enolfc** [comment]
> > Missing an ops project that we can reference here
> 
> For which purpose exactly?

We need the project id in OpenStack to be able to map it to a VO (ops) so cloud-info-provider can extract all the information

**Comment by sebastian-luna-valero** [comment]
xref: https://github.com/tdviet/fedcloudclient/pull/179

**Pull request merged**